roles/keystone/tasks/main.yml

```diff
@@ -56,6 +57,12 @@
     - 5001
     - 35358
 
+- name : change keystone log file ownership
+  file: path={{ item }} mode=0644 owner=keystone group=keystone state=file
+  with_items:
+    - /var/log/keystone/keystone-all.log
+    - /var/log/keystone/keystone-manage.log
```

Comment:
If these files do not exist when this task is ran (like say the initial deployment), a state of `file` will not create them. It'd probably be best to use the `touch` state, which will create the files if they don't exist and set the permissions correctly. If the file does exist, the permissions will be adjusted and the file will be "touched" just as if using `touch` from the command line.